scripts/run_node.sh

```diff
@@ -91,9 +12,15 @@ then
     echo "If you wish to customize the directory, select 'n' and re-run the script"
     echo "with the '--data_dir' argument."
     echo "Do you want to continue? [Y/n]"
-    read -r CONT
+    
+    if [[ "$PROMPTS" = true ]]
+    then
+        read -r CONT
+    else
+        CONT='y'
+    fi
 
-    if [[ "$CONT" == 'n' ]]
+    if [[ "$CONT" -eq 'n' ]]
```

Comment:
I don't thinks this works as intended :/

Reply:
+1, it should be = or == (or match it to [Nn])